common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -323,22 +312,7 @@ private void parseCharacterFromCharacterMenu(ItemStack characterInfoItem, ItemSt
         }
         ClassType classType = ClassType.fromName(className);
 
-        Map<ProfessionType, Integer> levels = new HashMap<>();
```

Comment:
Is the idea to restore this functionality in the ProfessionModel?

Reply:
It's been already done

Reply:
Nice, I must have just missed it. I'll let you continue working on the draft now, and stop pestering you until you are ready for review. :)

Reply:
> Nice, I must have just missed it. I'll let you continue working on the draft now, and stop pestering you until you are ready for review. :)

Now is the time :)